src/client/Microsoft.Identity.Client/AppConfig/PopAuthenticationConfiguration.cs

```diff
@@ -95,5 +95,14 @@ public PoPAuthenticationConfiguration(Uri requestUri)
         /// or as part of the AuthorityInfo header associated with 200 response. Set it here to make it part of the Signed HTTP Request part of the POP token.
         /// </summary>
         public string Nonce { get; set; }
+
+        /// <summary>
+        /// Allows app developers to bypass the creation of the SingedHttpRequest envelope by setting this property to true.
+        /// App developers can use a package like Microsoft.IdentityModel.Protocols.SignedHttpRequest to later create and sign the envelope. 
+        /// </summary>
+        /// <remarks>
+        /// If set to true, you do not need to implement the <see cref="IPoPCryptoProvider.Sign(byte[])"/> method when using custom keys. 
+        /// </remarks>
+        public bool DoNotSignHttpRequest { get; set; } = false;
```

Comment:
I think this should be positive. Better readability.
```suggestion
        public bool SignHttpRequest { get; set; } = true;
```